appveyor.yml

```diff
@@ -1,19 +1,21 @@
 version: '{build}'
 
+image: Visual Studio 2017
+
 platform:
   - x64
 
 services:
   - mssql2016
 
-shallow_clone: true
+clone_depth: 1
 
 cache:
-  - node_modules
```

Comment:
keep `node_modules` as well

Reply:
caching node_modules isn't safe, I would suggest caching only the npm cache

Reply:
As long as we are caching modules some where I have no problem, main goal is fast build startup